### PR TITLE
Stack view

### PR DESCRIPTION
**Pull request by t-heinkel**

### CODE REVIEW

**Comment by femto-code** [comment]
Ok

**Comment by t-heinkel** [comment]
Nice

File: apps/NewsFeedReader/qml/feedView.qml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    id: feed
+
+    Rectangle{
+        anchors.fill: parent
+        Text {
+            id: test
+            text: qsTr("Look at all my fancy Feeds")
+            anchors.centerIn: parent
+        }
+
```
Comment (femto-code) [inline apps/NewsFeedReader/qml/feedView.qml:14]: So this QML Stackview element is the place supposed to contain the XMLListModel, is that correct?
In order for me to implement this I need your changes in my branch. So either
1.) let's merge in main (and then I create a new branch from there)
2.) I create a new branch based on your branch StackView (which would it make a little more complicated 🤣 )

Comment (t-heinkel) [reply apps/NewsFeedReader/qml/feedView.qml:14]: Exactly.
Ok, then I'll finish the draft so we can merge in main. No need to make it more complicated 😄

**Pull request merged**